CHANGELOG.rst

```diff
@@ -15,6 +15,10 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Enumeration of product items increases over all productGroups in Offer PDF (`#562 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/562>`_)
 
+* Tax cost for offers outside of germany is set to 0 during offer creation (`#575 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/575>`_)
+
+* Tax cost for offers outside of germany is set to 0 in Offer PDF (`#575 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/575>`_)
+
```

Comment:
Yes, since this PR only changes the tax cost for newly created offers and in the offer pdf creation. 
This means that offers created before this PR show a taxation cost in the offer overview in the portlet.

Reply:
Both entries point to #575 😄

Reply:
Alrighty will do 👍